docs/UserGuide.md

```diff
@@ -290,6 +290,17 @@ Repository's Location | The `GitHub URL` or `Disk Path` to the git repository e.
 
 <sup>* **Multi-value column**: multiple values can be entered in this column using a semicolon `;` as the separator.</sup>
 
+#### `groups-config.csv`
+
+Optionally, you can provide a `groups-config.csv`(which should be in the same directory as `repo-config.csv` file) to provide more details about any groupings of files in the repository. It should contain the following columns:
+
+Column Name | Explanation
+----------- | -----------
+Group Name | Name of the group e.g.,`test`
+Globs | The list of file path globs to include for specified group. e.g.,`**/test/*;**.java`
+
+Note that each file can only have 1 group. If more than 1 group is specified for a given file, the latter group is set for the file.
```

Comment:
This sentence is misleading. It sounds like each csv file can have only one group.

Reply:
I've updated the User Guide for `groups-config.csv`, hopefully it is clearer now.